_database_api/status.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Instance Status
+layout: default
+---
+
+# Instance Status
+
+--- row ---
+
+## Get Database Instances Status
+
+--- row ---
+
+`GET https://$DB_API_URL/api/databases/[:database_id]/instances_status`
+
+{:.table}
+| field | type   | description                              |
+| --    | --     | --                                       |
+| id    | string | Instance ID                              |
+| type  | string | Instance type                            |
+| status| string | Instance status                          |
+| role  | string | Instance role in the cluster             |
+
+The request provides information about the instances of a given database.
+Type is about the instance type, so either "db-node" or "gateway" node. An additional service type "utility" can exist.
```

Comment:
The sentence "An additional service type "utility" can exist." is cryptic. If I'm a Scalingo customer and I read this, I don't think I would be satisfied by this. The "utility" type is used for Redis Sentinel andI think we should write that here.